external-crates/move/crates/move-binary-format/src/file_format_common.rs

```diff
@@ -131,8 +163,24 @@ pub enum SerializedType {
 #[repr(u8)]
 #[derive(Clone, Copy, Debug)]
 pub enum SerializedNativeStructFlag {
-    NATIVE                  = 0x1,
-    DECLARED                = 0x2,
+    NATIVE   = 0x1,
+    DECLARED = 0x2,
+}
+
+#[rustfmt::skip]
+#[allow(non_camel_case_types)]
+#[repr(u8)]
+#[derive(Clone, Copy, Debug)]
+pub enum SerializedEnumFlag {
+    DECLARED = 0x1,
```

Comment:
Nit, should this be `0x2` While silly, keeps it in line with `SerializedEnumFlag`

Reply:
Don't have a strong opinion either way -- happy to make it 0x2 :)